ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/datepicker.html

```diff
@@ -41,6 +41,7 @@
             disabled="${!datePicker.enabled}"
             readonly="${datePicker.readOnly}"
             required="${datePicker.required}"
+            value="${datePicker.default}"
```

Comment:
What is the default value of `default` attribute if nothing is set in authoring ? Shouldn't you check for not empty here ?